src/app/components/todo-list/todo-list.component.html

```diff
@@ -1,50 +0,0 @@
-
-<input type="text" class="todo-input" placeholder="What needs to be done"
-       [(ngModel)]="todoTitle" (keyup.enter)="addToDo()">
-
-<div class="todo-item" *ngFor="let todo of todosFiltered()"
```

Comment:
don't do this - todosFiltered(). Make some variable, write a value to it and iterate through it.